pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -416,13 +415,13 @@ func newKWebUIForCR(cr *hcov1alpha1.HyperConverged) *kwebuis.KWebUI {
 			Labels: labels,
 		},
 		Spec: kwebuis.KWebUISpec{
-			Version:                         "latest",                             // TODO: image tag name, use Version ; https://github.com/kubevirt/hyperconverged-cluster-operator/pull/22/files
-			RegistryUrl:                     "",                                   // TODO: use ContainerRegistry  ; https://github.com/kubevirt/hyperconverged-cluster-operator/pull/22/files
-			RegistryNamespace:               "",                                   // keep blank, already in ContainerRegistry
+			Version:                         "",                                   // will be equal to the web-ui-operator version (image tag)
```

Comment:
what do you think about not setting these at all and just having

```
kwebuis.KWebUISpec{
    OpenshiftMasterDefaultSubdomain: ...,
    PublicMasterHostname: ...,
}
```

Reply:
You can set Version, Registry, ect.. in the web-ui operator if they are not set on the web-ui CR.  Example in [CDI](https://github.com/mhenriks/containerized-data-importer/blob/master/pkg/operator/controller/controller.go#L498).

Reply:
Empty strings removed from here.
The web-ui-operator sets missing values based on ENV variables - https://github.com/kubevirt/web-ui-operator/pull/62/files